### ISSUE TITLE
ora/csv_util_pkg.pkb function clob_to_csv: quoted newline characters?

### ISSUE THREAD

**Issue by u797595** [mortenbra/alexandria-plsql-utils#87]
Does function clob_to_csv ( orc/csv_util_pkg.pkb ) support quoted newline characters? (LF|CRLF,  \n|\r\n)

https://datatracker.ietf.org/doc/html/rfc4180#page-2

"escaped = DQUOTE *(TEXTDATA / COMMA / CR / LF / 2DQUOTE) DQUOTE"

**Comment by mortenbra** [comment]
No, csv_util_pkg.clob_to_csv does not support quoted newline characters.

You might check out apex_data_parser.parse (see https://docs.oracle.com/en/database/oracle/apex/23.1/aeapi/PARSE-Function.html#GUID-B815CF74-C469-4F78-9433-643D1339E930 ) which is probably more standards-compliant.

**Comment by anja8080** [comment]
Hi,

i think have the same problem with line breaks in quoted comments. actually your packages identifies this as a new column.  but it's just a newline in a quoted comment. 
i would like to refere to this topic, where someone shared a workaround. but sadly it's not available anymore.
Is there a chance to download this code somewhere?

https://code.google.com/archive/p/plsql-utils/issues/27
